pkg/daemon/discover/discover.go

```diff
@@ -378,3 +383,102 @@ func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 	logger.Infof("available devices: %+v", devices)
 	return devices, nil
 }
+
+//finds all the lvm devices in the node which are not mapped to any process and deactivates them.
+func releaseLVMDevices(context *clusterd.Context) error {
+
+	logger.Info("checking for unreleased lvm devices")
+	var op string
+	var err error
+
+	//Get all the logical volumes.
+	if op, err = context.Executor.ExecuteCommandWithCombinedOutput(false, "", "lvdisplay", "-c"); err != nil {
```

Comment:
Why are we not using `--reportformat json` and instead parsing text?

Reply:
@BlaineEXE : I'll update the code to use  `--reportformat json`. Wasn't aware about this. Thanks.